molgenis/molgenis-emx2
# docs(catalogue): update researcher harmonisation overview docs

What are the main changes you did:
- explain what you changed and essential considerations.

how to test:
- see here: https://preview-emx2-pr-4242.dev.molgenis.org/apps/docs/#/catalogue/cat_researcher?id=find-variables

-chnages can be made and commited directly in online github branch here; https://github.com/molgenis/molgenis-emx2/blob/chore/update-researcher-docs/docs/catalogue/cat_researcher.md

todo:
